### DISCUSSION TITLE
"brew tap" does not show anything

### DISCUSSION THREAD

**Discussion by quackerex** [Homebrew/discussions#5594] [Everyday usage] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.3.19-103-g170a7ea
ORIGIN: https://github.com/Homebrew/brew
HEAD: 170a7ea56e14c123df912855171ed5a9a5f76890
Last commit: 6 hours ago
Core tap JSON: 07 Sep 22:54 UTC
Core cask tap JSON: 07 Sep 22:54 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
HOMEBREW_SORBET_RUNTIME: set
Homebrew Ruby: 3.3.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.3.4_1/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 8.7.1 => /usr/bin/curl
macOS: 14.6.1-arm64
CLT: 15.3.0.0.1.1708646388
Xcode: 15.4
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

According to the docs it should show the default taps: homebrew/core and homebrew/cask 

<img width="634" alt="Screenshot" src="https://github.com/user-attachments/assets/58558e56-5203-4639-8123-b317f345506c">

When I run `brew tap` it does not show anything.
<img width="220" alt="Screenshot" src="https://github.com/user-attachments/assets/a471e483-4fe0-4b4b-b23a-c3b65fc6f226">

**Comment by ZhongRuoyu** [comment, marked as answer]
What https://docs.brew.sh/Taps#the-brew-tap-command suggests is that `brew tap` lists all taps that are installed locally on your system. The official `homebrew/core` and `homebrew/cask` taps may not necessarily be tapped because `brew` installs formulae via API by default. (They used to be tapped by default but that was changed [quite a while ago](https://brew.sh/2023/02/16/homebrew-4.0.0/).)

**Reply by quackerex** [reply]
I want to untap from homebrew/cask(to avoid using casks entirely). I assume it's no longer possible?

**Reply by carlocab** [reply]
You do not need to untap homebrew/cask because you don't have homebrew/cask untapped.

**Comment by BoolPurist** [comment]
Then the documentation under [this site](https://docs.brew.sh/Taps) should be adjusted because it also confused me too with command `brew tap` not outputing anything.

**Reply by p-linnane** [reply]
Sounds good. Please open a PR and we will review it. Thanks!

**Reply by BoolPurist** [reply]
I opened a [PR](https://github.com/Homebrew/brew/pull/21099) for updating the documentation about that.
